[Documentation] Modifications sur les parties frontend et backend.

Jars-of-jam-Scheduler opened the pull request:
---> Quelques modifications sur les commandes Ubuntu
---> Ajout des commandes Fedora
---> Ajout de la commande gulp build (jusque-là, il n'était pas clairement demandé à l'utilisateur de la saisir, alors qu'elle est nécessaire s'il veut pouvoir consulter le site en local)
---> Commandes commentées (l'utilisateur doit-il être sudo ou pas, l'env. Python doit-il être activé ou pas, etc.)

Jars-of-jam-Scheduler committed:
---> Quelques modifications sur les commandes Ubuntu
---> Ajout des commandes Fedora
---> Ajout de la commande gulp build (jusque-là, il n'était pas clairement demandé à l'utilisateur de la saisir, alors qu'elle est nécessaire s'il veut pouvoir consulter le site en local)
---> Commandes commentées (l'utilisateur doit-il être sudo ou pas, l'env. Python doit-il être activé ou pas, etc.)

Jars-of-jam-Scheduler committed:
Rétablissement de la commande d'installation de nodejs-legacy

gustavi commented on doc/source/install/frontend-install.rst:
```diff
@@ -164,6 +164,12 @@ Gulp se lance avec ``npm run gulp -- [tâche]`` où ``[tâche]`` est la tâche 
 Générer les fichiers avec ``build``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Saisissez la commande suivante (elle vous permet de générer les fichiers Web du site pour vous permettre de consulter Zeste de Savoir sur votre navigateur, en local) :
```
« fichiers Web » c'est pas terrible

Jars-of-jam-Scheduler replied:
Pourquoi ?

gustavi commented on doc/source/install/backend-linux-install.rst:
```diff
@@ -68,9 +73,10 @@ Une fois dans votre environnement python (``source ../bin/activate`` si vous uti
+    pip install --upgrade -r requirements.txt -r requirements-dev.txt # Installe les dépendances Python (--upgrade = permet de mettre à jour les fichiers si et seulement si quelqu'un a mis à jour les fichiers "requirements") - Ne PAS être sudo, à faire dans le répertoire du clone de votre fork, activation de l'environnement Python requise
+    python manage.py migrate # Cf. "migrate" de Django (le faire à l'installation et à la mise à jour) - Ne PAS être sudo, à faire dans le répertoire du clone de votre fork, activation de l'environnement Python requise
+    python manage.py runserver # Démarre l'instance de ZdS - Ne PAS être sudo, à faire dans le répertoire du clone de votre fork, activation de l'environnement Python requise
+    deactivate
```
Aucun intérêt ici, autant rester dans l'environnement virtuel

Jars-of-jam-Scheduler replied:
Tu parles du `deactivate` ?

pierre-24 replied:
+1 pour que `deactivate` disparaisse :)

Jars-of-jam-Scheduler replied:
Yep je corrige ça !

gustavi commented on doc/source/install/backend-linux-install.rst:
```diff
@@ -68,9 +73,10 @@ Une fois dans votre environnement python (``source ../bin/activate`` si vous uti
-    pip install --upgrade -r requirements.txt -r requirements-dev.txt
-    python manage.py migrate
-    python manage.py runserver
+    pip install --upgrade -r requirements.txt -r requirements-dev.txt # Installe les dépendances Python (--upgrade = permet de mettre à jour les fichiers si et seulement si quelqu'un a mis à jour les fichiers "requirements") - Ne PAS être sudo, à faire dans le répertoire du clone de votre fork, activation de l'environnement Python requise
```
Les 3 lignes qui suivent me paraissent trop détaillées et redondantes sur sudo par exemple

Jars-of-jam-Scheduler replied:
Salut,

Justement c'est en grande partie pour écrire ces lignes que j'ai souhaité participer à la doc'. En effet, la plupart des soucis que j'ai rencontrés quand j'ai essayé d'installer l'environnement de travail venaient du fait que je n'activais pas l'env Python au bon moment (par exemple). Crois-moi, il est bien nécessaire d'apporter tous ces commentaires.

pierre-24 replied:
Je suis d'accord avec Lern-X, mais je trouve les commentaires "trop long". Soit tu résume en quelque mot (pour que ça tienne dans les 800px du layout), soit tu peux mettre les explications (résumées) en dessous :)

Jars-of-jam-Scheduler replied:
Ok, je vais faire ça (j'opterais plutôt pour les commentaires résumés en quelques mots) :).

Jars-of-jam-Scheduler committed:
Ce commit fait suite aux remarques de gustavi et de pierre-24.

---> Suppression de la commande "deactivate" ligne 79
---> Commentaires raccourcis (généralement déplacés en-dehors de la zone de code-source)
---> Trois petites corrections de français (oubli d'une ')', et deux oublis de '-')

pierre-24 commented:
Allez, zou :)
